comit/src/lnd.rs

```diff
@@ -351,6 +351,7 @@ impl LndConnectorAsReceiver {
 
         // Need to shortcut here until https://github.com/hyperium/hyper/issues/2171 or https://github.com/lightningnetwork/lnd/issues/4135 is resolved
         if response.status() == StatusCode::INTERNAL_SERVER_ERROR {
+            tracing::debug!("invoice not found");
```

Comment:
Is there enough context to know which invoice we are referring to when reading the logs?

Reply:
The function is instrumented by tracing so the answer is yes.

Reply:
Isn't it "the answer is yes if the context is properly set" ?

Reply:
If I understand it correctly, instrument will always use the function name and all or some arguments to the function as context.

Reply:
Correct, instrumenting a function automatically creates a span. See the documentation here: https://docs.rs/tracing/0.1.14/tracing/attr.instrument.html